cmd/bnsapi/handlers/handlers.go

```diff
@@ -578,19 +577,7 @@ func (h *CashBalanceHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 			JSONErr(w, http.StatusInternalServerError, http.StatusText(http.StatusInternalServerError))
 		}
 	} else {
-		// query all wallets
-
-		var offset []byte
-		if q.Get("offset")!= "" {
-			var err error
-			offset, err = ExtractAddress(q.Get("offset"))
-			if err != nil && !errors.ErrEmpty.Is(err) {
-				JSONErr(w, http.StatusBadRequest, "offset is in wrong format. send integer")
-				return
-			}
-		}
-
-		it := client.ABCIRangeQuery(r.Context(), h.Bns, "/wallets", fmt.Sprintf("%s:", offset))
+		it := client.ABCIPrefixQuery(r.Context(), h.Bns, "/wallets", []byte{})
```

Comment:
When I execute `curl -s -vvv slim:8001/cash/balances` I get
```
*   Trying 192.168.1.46:8001...
* TCP_NODELAY set
* Connected to slim (192.168.1.46) port 8001 (#0)
> GET /cash/balances HTTP/1.1
> Host: slim:8001
> User-Agent: curl/7.66.0
> Accept: */*
> 
* Mark bundle as not supporting multiuse
< HTTP/1.1 500 Internal Server Error
< Content-Type: application/json; charset=UTF-8
< Date: Wed, 22 Apr 2020 09:38:53 GMT
< Content-Length: 45
< 
{ [45 bytes data]
* Connection #0 to host slim left intact
```
on the client side and the following on the server side:
```
dave@slim bns(fix-cash)$ TENDERMINT=https://bnsapi.cluster-mainnet.iov.one:443/ HTTP=0.0.0.0:8001 bnsapi
dev handlers.go:598: cash balance ABCI query: response: decode response: invalid character '<' looking for beginning of value
```

Similarly, `curl -s -vvv 'http://slim:8001/username/resolve/thematrix*iov'` yields the following on the server:
```
dev username_handlers.go:89: account ABCI query: response: decode response: invalid character '<' looking for beginning of value
```